### PR TITLE
Switch to Circle CI

### PR DESCRIPTION
**Pull request by willdurand**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+defaults: &defaults
+  working_directory: ~/addons-scanner-utils
+  docker:
+    # This is the NodeJS version we run in production.
+    - image: circleci/node:10
+
+defaults-next: &defaults-next
+  working_directory: ~/addons-scanner-utils
+  docker:
+    # This is the NodeJS version we run in production.
```
Comment (muffinresearch) [inline .circleci/config.yml:15]: Presumably we're not running both 10 and 12 at the same time?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # addons-scanner-utils
 
-[![Build Status](https://travis-ci.com/mozilla/addons-scanner-utils.svg?branch=master)](https://travis-ci.com/mozilla/addons-scanner-utils) [![codecov](https://codecov.io/gh/mozilla/addons-scanner-utils/branch/master/graph/badge.svg)](https://codecov.io/gh/mozilla/addons-scanner-utils) [![npm version](https://badge.fury.io/js/addons-scanner-utils.svg)](https://badge.fury.io/js/addons-scanner-utils)
+[![codecov](https://codecov.io/gh/mozilla/addons-scanner-utils/branch/master/graph/badge.svg)](https://codecov.io/gh/mozilla/addons-scanner-utils) [![npm version](https://badge.fury.io/js/addons-scanner-utils.svg)](https://badge.fury.io/js/addons-scanner-utils)
```
Comment (muffinresearch) [inline README.md:3]: Adding a circle badge might be nice? It's handy at a glance to know that everything is green.

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+      - run:
+          name: publish package
+          command: npm publish
+
+workflows:
+  version: 2
+  default-workflow:
+    jobs:
+      - test:
+          filters: # required since `release-tag` has tag filters AND requires `test`
```
Comment (muffinresearch) [inline .circleci/config.yml:101]: Comment is appreciated since lack of familiarity makes understanding this config much harder.

**Review by muffinresearch — APPROVED** [review]
r+wc lgtm

File: .circleci/config.yml (hunk 1 above)
Comment (willdurand) [reply .circleci/config.yml:15]: doh, wrong copy paste

File: README.md (hunk 2 above)
Comment (willdurand) [reply README.md]: Yep, thanks for pointing me to the docs. I thought we could generate a badge from the settings (which is how I get badges for private repos).

**Pull request merged**